### PR TITLE
Convert zoom/camera to use matrix

### PR DESCRIPTION
**Pull request by k88hudson**
Use a css matrix will make improving the origin stuff a lot easier; I wanted to do this step separately so the PR is smaller.

The way it works is pretty simple, 2D matrices are just 6 numbers:

```
[scaleX, 0, 0, scaleY, transformX, transformY]
```

To test, just try the project view (there should be no changes in UX)

### CODE REVIEW

File: src/pages/project/project.jsx [hunk 1]
```diff
@@ -63,8 +63,7 @@ var Project = React.createClass({
         if (state.params && state.params.project === this.state.params.project) {
           this.setState({
             selectedEl: state.selectedEl,
-            camera: state.camera,
-            zoom: state.zoom
+            matrix: state.matrix
           });
         }
       }
```
Comment (gvn) [inline src/pages/project/project.jsx:69]: Just remember that `player.jsx` will need updates in `webmaker-browser`...

Comment (k88hudson) [inline src/pages/project/project.jsx:69]: ok, PR is here: https://github.com/mozilla/webmaker-browser/pull/197

**Comment by gvn** [comment]
I'm not sure how this is still working, but looks like a comma is missing:

``` bash
SyntaxError: /Users/gsuntop/Documents/mozilla-code/webmaker-core/src/tests/spec.test.js: Unexpected token (89:8)
  87 |         transformOrigin: 'center',
  88 |         WebkitTransformOrigin: 'center'
> 89 |         zIndex: 5
     |         ^
  90 |       });
  91 |     });
  92 |   });
```

**Comment by gvn** [comment]
Oh, that's a test...derp. Still, good to fix.

Otherwise looks good!

**Comment by k88hudson** [comment]
oops, good catch!

**Comment by k88hudson** [comment]
@gvn ok updated

**Comment by gvn** [comment]
Did you forget to push it?

**Comment by k88hudson** [comment]
@gvn oops yep

**Comment by gvn** [comment]
R+!

**Pull request merged**